testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -215,30 +213,30 @@ def test_empty_ValueError(self):
     def test_nonsense_TypeError(self):
         assert_raises(TypeError, Merge, ['1', 2])
 
-    def test_emptyAG_ValueError(self):
-        u = self.universes[0]
+    def test_emptyAG_ValueError(self, universes):
+        u = universes[0]
         a = AtomGroup([], u)
         b = AtomGroup([], u)
 
         assert_raises(ValueError, Merge, a, b)
 
 
-class TestMergeTopology(TestCase):
+class TestMergeTopology(object):
     """Test that Merge correct does topology"""
 
-    @dec.skipif(parser_not_found('DCD'),
-                'DCD parser not available. Are you using python 3?')
-    def setUp(self):
-        self.u = MDAnalysis.Universe(PSF, DCD)
-        self.u2 = MDAnalysis.Universe(merge_protein)
+    @staticmethod
+    @pytest.fixture()
+    def u():
+        return MDAnalysis.Universe(PSF, DCD)
 
-    def tearDown(self):
-        del self.u
-        del self.u2
+    @staticmethod
+    @pytest.fixture()
+    def u2():
```

Comment:
This is the same as `TestCaping.u2`. You can move `TestCaping.u1`, `TestCaping.u2`, and `TestCaping.u3` in the module scope instead of the scope of the scope of the `TestCaping` class. Then you can use the same fixture thoughout the file and we know they correspind to the same things. Do not forget the renaming I asked above, though.